src/w3c/templates/show-people.js

```diff
@@ -7,19 +7,20 @@ export default (conf, name, items = []) => {
   return results;
 
   function getItem(p, i) {
+    const personName = html`${{html: p.name}}`;
     const editorid = p.w3cid ? parseInt(p.w3cid, 10): null;
     const dd = html`<dd class='p-author h-card vcard'
       data-editor-id='${editorid}'></dd>`;
       const span = document.createDocumentFragment();
     const contents = [];
     if (p.mailto) {
       contents.push(html`<a class='ed_mailto u-email email p-name'
-        href='${`mailto:${p.mailto}`}'>${p.name}</a>`);
+        href='${`mailto:${p.mailto}`}'>${personName}</a>`);
```

Comment:
`${[p.name]}` should be enough to treat it as HTML.

https://viperhtml.js.org/hyperhtml/documentation/#essentials-8

>if it's an array of strings, they will be injected as explict opt-in for HTML